source/v2-scheduler.rst

```diff
@@ -0,0 +1,139 @@
+.. _psqladm-scheduler:
+
+==========================================================
+ProxySQL and Percona Scheduler Admin
+==========================================================
+
+Implemented in ProxySQL 2.3.2-1, the Percona Scheduler Admin (percona-scheduler-admin) utility configures **Percona XtraDB Cluster** nodes into *ProxySQL*. 
+
+The Percona Scheduler Admin works with ProxySQL 2.3.0.  
+
+The utility has an external scheduler which is capable of the following operations:
+
+* Segment aware failover mechanism
+* Parallel query execution on nodes results in faster failover or fallback
+* Supports fallback
+* Supports promoting a reader to primary with the following conditions are met:
+    * wsrep_on=OFF
+    * wsrep_sst_donor_rejects_queries is set when, for example, connections are rejected due to an ongoing state transfer
+* Supports a retry of operations in case of network glitches
+
+This tool can be used to automatically perform a failover due to node failures, service degradation, or maintenance. 
+
+Please log ProxySQL Admin bug reports here: https://jira.percona.com/projects/PSQLADM.
+
+.. note::
+
+    The Percona Scheduler Admin utility has different features than :ref:`v2-config`. You cannot use options from one utility in the other utility. Mixing the options may cause unintended results.
+
+.. rubric:: Prerequisites
+
+The following are the prerequisites for using the Percona Scheduler Admin:
+
+* The mysql client and my_print_defaults utility must be installed on the system.
+    
+* The ProxySQL and Percona XtraDB Cluster should be up and running.
+
+
+.. rubric:: The percona-scheduler-admin usage info
+
+.. sourcecode:: bash
+
+   percona-scheduler-admin [ options ]
+   
+.. seealso:: :ref:`psa-options` for examples of common operations.
+
+You must include at least one options. The options are either independent, do not require other options, or are dependent on other options. 
```

Comment:
The options are either independent, do not require other options, -> I would add either "and" here or ()

"The options are either independent and do not require other options,"
or 
"The options are either independent (do not require other options),"

Reply:
Rewritten.